### ISSUE TITLE
Giving airflow:airflow permission to pvc folder

### ISSUE THREAD

**Issue by shanky221341** [airflow-helm/charts#86]
I have mounted EFS PVC at /jsons folder using extra volume config in values.yaml file. I want to write some something here which will be used by /opt/airflow/dags folder. 

It gets mounted correctly, and dags can read files from here, but when tries to write something it gets "permission denied error". Since default permission is root:root when pvc mount is done.

I know if we change fsGroup this may work according to https://kubernetes.io/docs/tasks/configure-pod-container/security-context/

But, is there any way I can change user and group of any folder inside webserver/scheduler/worker pods using values.yaml file or any other means?

**Comment by adamd01** [comment]
@shanky221341  I had this exact same issue today, this is the way I managed to get it working:

For web and scheduler pods this worked in values.yaml:

```
scheduler:
  securityContext:
    fsGroup: 50000

...

web:
  securityContext:
    fsGroup: 50000
...
```

For the worker though it was different because I'm using the k8s executor. I managed to set that using environment variable config - it seems although only some are documented actually anything in airflow.cfg can be set from environment variables (from what I've seen)

So in values.yaml
```
airflow:

  executor: KubernetesExecutor

  config:
    AIRFLOW__KUBERNETES__RUN_AS_USER: "50000"
    AIRFLOW__KUBERNETES__FS_GROUP: '50000'
...
```

This might also work for the web & scheduler too, but I've got both in just in case

**Comment by shanky221341** [comment]
@adamd01 Thanks for the detailed answer. I also noticed that fsgroup option is available in values.yaml file and I did.

securityContext:
    runAsUser: 50000
    runAsGroup: 50000
    fsGroup: 50000  


for web pod only to check. There were no errors anywhere but for some reason it did not work for me.

But following worked just now inside scheduler section of values.yaml:

  extraInitContainers:
    - name: volume-mount-hack
      image: busybox
      command: ["sh", "-c", "chown -R 50000:50000 /jsons folder"]
      volumeMounts:
        - mountPath: /jsons folder
          name: dags-data

Surprisingly user and group got changed in web and worker pods too without explicitly doing above in their respective sections or deployment.yaml.
